src/components/Camera/hooks/usePrepareStoreAndNavigate.js

```diff
@@ -65,6 +65,11 @@ export async function savePhotosToCameraGallery(
         // the EXIF metadata of these photos, once we retrieve a location.
         onEachSuccess( savedPhotoUri );
       } catch ( cameraRollSaveError ) {
+        if ( cameraRollSaveError.message.match( /No space left on device/ )
+          || cameraRollSaveError.message.match( /The operation couldn’t be completed/ ) ) {
+          console.log( "cameraRollSaveError: ", cameraRollSaveError );
+          return;
+        }
```

Comment:
Remove the `console.log` statement since that's only for dev purposes. Also, you're silently suppressing a real error here. Don't you want to alert the user instead? If you really don't, please leave a comment explaining why this error is ok to ignore.

Reply:
Better, but you should never show the user the actually error object. It will look like a bunch of incomprehensible code nonsense. The `message` attribute is better, but probably still too jargon-y and definitely not localized. Instead, tell the user what actually went wrong with globalized text that a layperson would understand. You're catching two separate kinds of errors here, which probably requires two separate messages to the user.